internal/kibana/client.go

```diff
@@ -80,6 +80,11 @@ func NewClient(opts ...ClientOption) (*Client, error) {
 	return c, nil
 }
 
+// Get client host
+func GetAddress(c Client) string {
+	return c.host
+}
```

Comment:
Nit. It'd be probably better to make this a member of the Client type.
```suggestion
// Get client host
func (c *Client) Address() string {
	return c.host
}
```